sql/builtin_functions.cc

```diff
@@ -2495,6 +2495,79 @@ BoundFunction length(const std::vector<BoundSqlExpression> & args)
 
 static RegisterBuiltin registerLength(length, "length");
 
+static char32_t getChar32(const string & in, int & idx) {
```

Comment:
That utf8 decoding code looks buggy to me... I wrote a version a while back and it's a bit more complicated then that. For starter, it can have up to 6 bytes if I remember correctly. See:
https://github.com/EricRobert/rig-string/blob/master/include/utf8.hpp#L123